cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/CardShowTakenPictureViewImagesAdapter.java

```diff
@@ -141,15 +176,32 @@ public int getItemCount() {
         void updateView(CardShowTakenImage cardShowTakenImage) {
             this.mServiceInspectionsFormFilledRecycleItemBinding.setCardStateEnum(mView.getActualCardState());
 
-            setImageBitmapToImageView(this.mServiceInspectionsFormFilledRecycleItemBinding.cardShowTakenPictureViewGeneralCircularImageView, cardShowTakenImage, 8);
+            setImageBitmapToImageView(this.mServiceInspectionsFormFilledRecycleItemBinding.cardShowTakenPictureViewGeneralCircularImageView,
+                    cardShowTakenImage, 8);
 
             this.mServiceInspectionsFormFilledRecycleItemBinding.setCardShowTakenImage(cardShowTakenImage);
             this.mServiceInspectionsFormFilledRecycleItemBinding.cardShowTakenPictureViewGeneralCircularImageView
                     .setOnClickListener(
                             v -> mView.showPreviewPicDialog(cardShowTakenImage)
-            );
+                    );
             this.mServiceInspectionsFormFilledRecycleItemBinding.executePendingBindings();
         }
+
+        @Override
+        public void onItemSelected() {
+            VibratorUtils.vibrate(mView.getContext(), 400);
+            mServiceInspectionsFormFilledRecycleItemBinding.cardShowTakenPictureContainerConstraintLayout.setAlpha(0.75f);
+            mServiceInspectionsFormFilledRecycleItemBinding.cardShowTakenPicturesItemDeleteIconContainerCardView.setVisibility(View.GONE);
+        }
+
+        @Override
+        public void onItemClear() {
+            mServiceInspectionsFormFilledRecycleItemBinding.cardShowTakenPictureContainerConstraintLayout.setAlpha(1);
+            mServiceInspectionsFormFilledRecycleItemBinding.cardShowTakenPicturesItemDeleteIconContainerCardView
+                    .setVisibility(mView.getActualCardState() == CardShowTakenPictureStateEnum.EDIT ? View.VISIBLE: View.GONE);
```

Comment:
create a method to ternary condition to become more explanatory

Reply:
Created